core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2063,6 +2063,7 @@ class SparkContext(config: SparkConf) extends Logging {
    * @param jobId the job ID to cancel
    * @throws InterruptedException if the cancel message cannot be sent
    */
+  @throws(classOf[InterruptedException])
```

Comment:
I think these need to be reverted too; we don't want to introduce checked exceptions